webwolf/pom.xml

```diff
@@ -66,6 +66,16 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
+                    <artifactId>log4j-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Why do we need this specific exclusion? We are not using log4j (you need to include a specific spring-boot starter).

Reply:
It is not really required for the update to the latest spring boot version. I added the exclusion since the owasp dependency check reports critical issues on these two libraries. And since they are not even used, why not remove them.